packages/common/sagas/common/share.js

```diff
@@ -210,18 +211,26 @@ export function* updateSharedItemFromServer({ payload: { itemId } }) {
 // 3. Share the system keyPair item to the new members
 export function* shareItemBatchSaga({
   payload: {
-    data: { itemIds = [], members = [] },
+    data: { itemIds = [], members = [], teamIds = [] },
   },
 }) {
   try {
     yield put(updateGlobalNotification(SHARING_IN_PROGRESS_NOTIFICATION, true));
     // const currentTeamId = yield select(currentTeamIdSelector);
     const items = yield select(itemsBatchSelector, { itemIds });
+    const teamsMembers = yield select(teamsMembersFullViewSelector, {
+      teamIds,
+    });
```

Comment:
please, add `|| []` to avoid crushes in runtime.

Reply:
`teamsMembersFullViewSelector` has such a check